src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.Default.cs

```diff
@@ -122,6 +122,39 @@ public void Default_SliceNegativeLength()
             Assert.Throws<ArgumentOutOfRangeException>("length", () => buffer.Slice(buffer.Start, -1L));
         }
 
+        [Fact]
```

Comment:
Can you share the code coverage numbers. Let's make sure all the new branches are covered (and if not, add more tests).

Reply:
How do people generally share this? Can I just upload the result of the code coverage run here? In any case, I added only 1 new branch in this patch and code coverage is telling me that both branches are covered by the tests. 

<img width="830" alt="CodeCoverage" src="https://user-images.githubusercontent.com/10480119/60047590-ee536800-967e-11e9-9117-048868b00f36.png">

Reply:
Yep, uploading the zip containing the code coverage works.

Looks like we have other branches too. Like in `Slice(long start, SequencePosition end)`

Reply:
Welp, it looks like github doesn't like zips

Reply:
"\\scratch2\scratch\prgovi\ROS_Slice_CodeCoverage.zip"